sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -303,11 +309,24 @@ annotated_ptr(T *Ptr, const properties<PropertyListT> &P = properties{}) noexcep
 ----
 |
 Constructs an `annotated_ptr` object. Does not allocate new storage. The
-underlying pointer is initialized with `Ptr`. `P` is used to specify the
-`PropertyListT` type on the class.
+underlying pointer is initialized with `Ptr`.
+
+The new property set `PropertyListT` contains all properties in `P`.
+If there were duplicate property in `P`, the value of this property must be same.
+
+// --- ROW BREAK ---
+a|
+[source,c++]
+----
+template<typename T, typename... PropertyValueTs>
+annotated_ptr(T *Ptr, const PropertyValueTs... props) noexcept;
+----
+|
+Constructs an `annotated_ptr` object. Does not allocate new storage. The
+underlying pointer is initialized with `Ptr`.
 
-The new property set `PropertyListT` must contain all properties from `P`,
-and if any common property takes a value, the value must be the same.
+The new property set `PropertyListT` contains all properties listed in `props`.
```

Comment:
Did spec reviewers request that this wording be changed?

Reply:
yes